freqtrade/rpc/api_server/api_schemas.py

```diff
@@ -7,6 +7,11 @@
 from freqtrade.enums import OrderTypeValues
 
 
+def dtype_string(dtype):
+    result = str(dtype.dtype) if type(dtype) == "datetime" else type(dtype).__name__
```

Comment:
i'd not do this as strings ... `type(dtype)` - but with `isinstance` instead.

Reply:
refactored (byusing `isnull()`) and rebased it

I'm not sure if it's still needed, but I guess it doesn't hurt if json_encoder is a little bit more robust?